packages/ember-data/lib/adapters/rest-adapter.js

```diff
@@ -838,3 +838,13 @@ function endsWith(string, suffix) {
     return string.endsWith(suffix);
   }
 }
+
+Ember.defineProperty(RESTAdapter, 'maxUrlLength', {
```

Comment:
Might have needed something more for IE8, but I think we are gonna drop it today